src/Adapter/Product/CommandHandler/AddCustomizationFieldHandler.php

```diff
@@ -28,56 +28,76 @@
 
 namespace PrestaShop\PrestaShop\Adapter\Product\CommandHandler;
 
-use CustomizationField as CustomizationFieldEntity;
+use CustomizationField;
```

Comment:
Just a side note: I wonder if we should use something like `use CustomizationField as LegacyCustomizationField` everytime we use legacy objectmodels This would clarify which objects we are using especially since we have some VOs which names match with the legacy objects

Reply:
in some other places I used CustomizationFieldDTO to mark it. (same idea, just reversed, instead of marking ObjectModel I marked the DTO)